### DISCUSSION TITLE
Installation geht nicht / SD Karte wir nicht erkannt 15.7.0

### DISCUSSION THREAD

**Discussion by pillemats** [jomjol/AI-on-the-edge-device#2950] [General]
### Issue

Hallo zusammen,

Ich habe das Problem das einer meiner ESP32 nicht mehr funktioniert hat.
Ich hatte die sd karte in vermutung.

da er nicht so wichtig war, habe ich einfach die komplette installation neu gemacht.
Also den webinstaller gestartet, installation gemacht, sd karte bespielt und rein.

leider blinkt der esp nur und die karte wird scheinbar nicht richtig erkannt...

ich habe nun schon 5 versch. karten getestet und sogar einen anderen ESP, aber ich habe überall das selbe problem.

Hat jemand eine idee wo ich noch ansetzen kann?

hier mal der log:

```
rst:0x1 (POWERON_RESET),boot:0x12 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:7664
load:0x40078000,len:15360
load:0x40080400,len:3744
entry 0x4008067c
I (27) boot: ESP-IDF 4.3.2 2nd stage bootloader
I (27) boot: compile time 20:02:47
I (27) boot: chip revision: 3
I (30) boot_comm: chip revision: 3, min. bootloader chip revision: 0
I (37) boot.esp32: SPI Speed      : 40MHz
I (42) boot.esp32: SPI Mode       : DIO
I (46) boot.esp32: SPI Flash Size : 4MB
I (51) boot: Enabling RNG early entropy source...
I (56) boot: Partition Table:
I (60) boot: ## Label            Usage          Type ST Offset   Length
I (67) boot:  0 nvs              WiFi data        01 02 00009000 00004000
I (75) boot:  1 otadata          OTA data         01 00 0000d000 00002000
I (82) boot:  2 phy_init         RF data          01 01 0000f000 00001000
I (89) boot:  3 ota_0            OTA app          00 10 00010000 001db000
I (97) boot:  4 ota_1            OTA app          00 11 001f0000 001db000
I (104) boot: End of partition table
I (109) boot_comm: chip revision: 3, min. application chip revision: 0
I (116) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=40014h (262164) map
I (225) esp_image: segment 1: paddr=0005003c vaddr=3ffb0000 size=040e8h ( 16616) load
I (232) esp_image: segment 2: paddr=0005412c vaddr=40080000 size=0beech ( 48876) load
I (253) esp_image: segment 3: paddr=00060020 vaddr=400d0020 size=1100f4h (1114356) map
I (680) esp_image: segment 4: paddr=0017011c vaddr=4008beec size=0e354h ( 58196) load
I (718) boot: Loaded app from partition at offset 0x10000
I (718) boot: Disabling RNG early entropy source...
I (730) cpu_start: Multicore app
I (730) quad_psram: This chip is ESP32-D0WD
I (730) esp_psram: Found 8MB PSRAM device
I (731) esp_psram: Speed: 40MHz
I (734) esp_psram: PSRAM initialized, cache is in low/high (2-core) mode.
W (741) esp_psram: Virtual address not enough for PSRAM, map as much as we can. 4MB is mapped
I (749) cpu_start: Pro cpu up.
I (752) cpu_start: Starting app cpu, entry point is 0x40082e0c
I (0) cpu_start: App cpu up.
I (1636) esp_psram: SPI SRAM memory test OK
I (1645) cpu_start: Pro cpu start user code
I (1645) cpu_start: cpu freq: 160000000 Hz
I (1645) cpu_start: Application information:
I (1646) cpu_start: Project name:     AI-on-the-edge
I (1650) cpu_start: App version:      v15.7.0
I (1654) cpu_start: Compile time:     Feb 17 2024 00:03:24
I (1660) cpu_start: ELF file SHA256:  71b36fda6e4066fa...
I (1665) cpu_start: ESP-IDF:          5.1.2
I (1669) cpu_start: Min chip rev:     v0.0
I (1672) cpu_start: Max chip rev:     v3.99 
I (1676) cpu_start: Chip rev:         v3.1
I (1681) heap_init: Initializing. RAM available for dynamic allocation:
I (1687) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
I (1692) heap_init: At 3FFBC688 len 00023978 (142 KiB): DRAM
I (1697) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
I (1703) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
I (1708) heap_init: At 4009A240 len 00005DC0 (23 KiB): IRAM
I (1714) esp_psram: Adding pool of 4082K of PSRAM memory to heap allocator
I (1722) spi_flash: detected chip: generic
I (1724) spi_flash: flash io: dio
I (1739) app_start: Starting scheduler on CPU0
I (1739) app_start: Starting scheduler on CPU1
I (1739) main_task: Started on CPU0
I (1749) esp_psram: Reserving pool of 40K of internal memory for DMA/internal allocations
I (1749) main_task: Calling app_main()
I (1749) MAIN: 



================ Start app_main =================
I (1859) MAIN: =================================================
I (1869) MAIN: ==================== Start ======================
I (1879) MAIN: =================================================
I (1879) MAIN: PSRAM size: 8388608 byte (8MB / 64MBit)
I (1889) MAIN: Total heap: 4380199 byte
I (1899) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (3899) gpio: GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2 
I (3899) cam_hal: cam init ok
I (3899) sccb: pin_sda 26 pin_scl 27
I (3899) sccb: sccb_i2c_port=1
I (3899) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (3939) camera: Detected camera at address=0x30
I (3939) camera: Detected OV2640 camera
I (3939) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4019) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4019) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4029) cam_hal: cam config ok
I (4029) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6109) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
I (6109) SDCARD: Basic R/W check started...
I (6129) SDCARD: Basic R/W check successful
I (6179) SNTP: TimeServer not defined, using default: pool.ntp.org
I (6189) SNTP: Configuring NTP Client...
I (6199) SNTP: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
I (6199) SNTP: time zone: +0100 Delta to UTC: 3600 seconds
I (6209) SNTP: The local time is unknown, starting with 1970-01-01 01:00:04
I (6219) SNTP: Once the NTP server provides a time, we will switch to that one
I (6239) MAIN: CPU frequency: 160 MHz
I (6239) OTA: Start CheckOTAUpdateCheck...
I (6259) OTA: SHA-256 for the partition table: : 9d4a2809ae90fb20709a8ca3dd3ac3462e1e73286bcd31b689060c91de06fa3a
I (6279) OTA: SHA-256 for bootloader: : 4a9573dafac5d5c79c43e5a356568332da3a99dd2e220da6ceeadc96d32708a2
I (6659) OTA: SHA-256 for current firmware: : 9a05b58889422bcde87ba674b3146abea61153f01a9043d02ab6712d64cd4a6f
I (6699) SDCARD: Folder/file presence check started...
E (6719) SDCARD: Folder/file check: File /html/version.txt not found
E (6729) HELPER: New System Status: 0x00000010
I (6739) MAIN: Tag: 'v15.7.0', Release: v15.7.0 (Commit: 0d0b018+), Date/Time: 2024-02-17 00:15, Web UI: ?
W (6749) MAIN: Failed to read file html/version.txt to parse Web UI version
W (6759) MAIN: Web UI version (?) does not match firmware version (0d0b018+)
W (6769) MAIN: Recommendation: Repeat installation using AI-on-the-edge-device__update__*.zip
I (6779) MAIN: Reset reason: Power-on event (or reset button)
I (6789) WLANINI: SSID: Cam
I (6789) WLANINI: Password: XXXXXXXX
I (6799) WLANINI: RSSIThreshold: 0
I (6809) MAIN: WLAN config loaded, init WIFI...
I (6819) WIFI: Automatic interface config --> Use DHCP service
I (6839) wifi:wifi driver task: 3ffd049c, prio:23, stack:6144, core=0
I (6849) wifi:wifi firmware version: 91b9630
I (6849) wifi:wifi certification version: v7.0
I (6849) wifi:config NVS flash: enabled
I (6859) wifi:config nano formating: enabled
I (6859) wifi:Init data frame dynamic rx buffer num: 16
I (6859) wifi:Init static rx mgmt buffer num: 5
I (6869) wifi:Init management short buffer num: 32
I (6869) wifi:Init static tx buffer num: 16
I (6879) wifi:Init tx cache buffer num: 16
I (6879) wifi:Init static rx buffer size: 1600
I (6879) wifi:Init static rx buffer num: 16
I (6889) wifi:Init dynamic rx buffer num: 16
I (6889) wifi_init: rx ba win: 16
I (6899) wifi_init: tcpip mbox: 32
I (6899) wifi_init: udp mbox: 6
I (6899) wifi_init: tcp mbox: 6
I (6899) wifi_init: tcp tx win: 5744
I (6909) wifi_init: tcp rx win: 5744
I (6909) wifi_init: tcp mss: 1440
I (6909) wifi_init: WiFi/LWIP prefer SPIRAM
I (6919) wifi_init: WiFi IRAM OP enabled
I (6919) wifi_init: WiFi RX IRAM OP enabled
I (6929) phy_init: phy_version 4780,16b31a7,Sep 22 2023,20:42:16
I (7009) wifi:mode : sta (08:d1:f9:6c:2d:08)
I (7009) wifi:enable tsf
I (7019) WIFI: Set hostname to: watermeter
I (7019) WIFI: Init successful
I (9429) wifi:new:<11,0>, old:<1,0>, ap:<255,255>, sta:<11,0>, prof:1
I (9429) wifi:state: init -> auth (b0)
I (10159) wifi:state: auth -> assoc (0)
I (11159) wifi:state: assoc -> init (2700)
I (11159) wifi:new:<11,0>, old:<11,0>, ap:<255,255>, sta:<11,0>, prof:1
W (11159) WIFI: Disconnected (39)
I (13589) wifi:new:<11,0>, old:<11,0>, ap:<255,255>, sta:<11,0>, prof:1
I (13589) wifi:state: init -> auth (b0)
I (14589) wifi:state: auth -> init (200)
I (14589) wifi:new:<11,0>, old:<11,0>, ap:<255,255>, sta:<11,0>, prof:1
I (14589) wifi:new:<1,0>, old:<11,0>, ap:<255,255>, sta:<1,0>, prof:1
I (14599) wifi:state: init -> auth (b0)
I (15599) wifi:state: auth -> init (200)
I (15599) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:1
W (15599) WIFI: Disconnected (2, Auth fail)
W (18029) WIFI: Disconnected (205)
I (20449) wifi:new:<11,0>, old:<1,0>, ap:<255,255>, sta:<11,0>, prof:1
I (20449) wifi:state: init -> auth (b0)
I (21449) wifi:state: auth -> init (200)
I (21449) wifi:new:<11,0>, old:<11,0>, ap:<255,255>, sta:<11,0>, prof:1
I (21449) wifi:new:<1,0>, old:<11,0>, ap:<255,255>, sta:<1,0>, prof:1
I (21459) wifi:state: init -> auth (b0)
I (22459) wifi:state: auth -> init (200)
I (22459) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:1
W (22459) WIFI: Disconnected (2, Auth fail)
W (24889) WIFI: Disconnected (205)
I (27029) MAIN: Device info: CPU cores: 2, Chip revision: 301
I (27039) MAIN: SD card info: Name: SDABC, Capacity: 14910MB, Free: 14878MB
I (29039) MAIN SERVER: Starting server on port: '80'
I (29039) MAIN SERVER: Registering URI handlers
I (29039) MAINCTRL: server_main_flow_task - Registering URI handlers
I (29039) OTA: Registering URI handlers
I (29039) GPIO: start GpioHandler
I (29049) GPIO: register GPIO Uri
I (29049) GPIO: server_GPIO - Registering URI handlers
E (29049) MAIN: Initialization failed. Flow task start aborted. Loading reduced web interface...
I (29069) main_task: Returned from app_main()
I (31339) WIFI: Connected to: Cam, RSSI: -63
I (34329) esp_netif_handlers: sta ip: 192.168.178.67, mask: 255.255.255.0, gw: 192.168.178.1
I (34329) WIFI: Assigned IP: 192.168.178.67
W (65539) wifi:[ADDBA]rx delba, code:39, delete tid:0
>
```

**Comment by SybexX** [comment]
dein Fehler >>>  W (6749) MAIN: Failed to read file html/version.txt to parse Web UI version
Auf der sd Karte im Ordner html ist die Datei version.txt nicht vorhanden.
Wenn du selber Kompelierst, findest du sie im Ordner code/sd-card/html.

**Comment by pillemats** [comment]
das hat geklappt! Danke dir! 

bei meinem anderen esp habe ich das selbe gemacht, aber ich komme hier nicht auf das webinterface, obwohl die sd karten daten gleich sind, bis auf der name in der ini und auch die feste ip adresse.

bei der inititalisierung ging die led an, somit sollte er sich ja mit dem wlan verbunden haben


EDIT: nach mehrmaligem neustart scheint es jetzt zu funktionieren, ohne änderungen vorzunehmen
EDIT2: hab mich zu früh gefreut... scheinbar war es nur ein cache. ich komme nicht aufs webif
hier mal der log


ets Jul 29 2019 12:21:46

rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:7664
load:0x40078000,len:15360
load:0x40080400,len:3744
entry 0x4008067c
I (27) boot: ESP-IDF 4.3.2 2nd stage bootloader
I (27) boot: compile time 20:02:47
I (27) boot: chip revision: 3
I (30) boot_comm: chip revision: 3, min. bootloader chip revision: 0
I (37) boot.esp32: SPI Speed      : 40MHz
I (42) boot.esp32: SPI Mode       : DIO
I (46) boot.esp32: SPI Flash Size : 4MB
I (51) boot: Enabling RNG early entropy source...
I (56) boot: Partition Table:
I (60) boot: ## Label            Usage          Type ST Offset   Length
I (67) boot:  0 nvs              WiFi data        01 02 00009000 00004000
I (75) boot:  1 otadata          OTA data         01 00 0000d000 00002000
I (82) boot:  2 phy_init         RF data          01 01 0000f000 00001000
I (89) boot:  3 ota_0            OTA app          00 10 00010000 001db000
I (97) boot:  4 ota_1            OTA app          00 11 001f0000 001db000
I (104) boot: End of partition table
I (109) boot_comm: chip revision: 3, min. application chip revision: 0
I (116) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=40014h (262164) map
I (225) esp_image: segment 1: paddr=0005003c vaddr=3ffb0000 size=040e8h ( 16616) load
I (232) esp_image: segment 2: paddr=0005412c vaddr=40080000 size=0beech ( 48876) load
I (253) esp_image: segment 3: paddr=00060020 vaddr=400d0020 size=1100f4h (1114356) map
I (680) esp_image: segment 4: paddr=0017011c vaddr=4008beec size=0e354h ( 58196) load
I (718) boot: Loaded app from partition at offset 0x10000
I (718) boot: Disabling RNG early entropy source...
I (730) cpu_start: Multicore app
I (730) quad_psram: This chip is ESP32-D0WD
I (731) esp_psram: Found 8MB PSRAM device
I (731) esp_psram: Speed: 40MHz
I (734) esp_psram: PSRAM initialized, cache is in low/high (2-core) mode.
W (741) esp_psram: Virtual address not enough for PSRAM, map as much as we can. 4MB is mapped
I (749) cpu_start: Pro cpu up.
I (752) cpu_start: Starting app cpu, entry point is 0x40082e0c
I (0) cpu_start: App cpu up.
I (1636) esp_psram: SPI SRAM memory test OK
I (1645) cpu_start: Pro cpu start user code
I (1645) cpu_start: cpu freq: 160000000 Hz
I (1645) cpu_start: Application information:
I (1646) cpu_start: Project name:     AI-on-the-edge
I (1650) cpu_start: App version:      v15.7.0
I (1654) cpu_start: Compile time:     Feb 17 2024 00:03:24
I (1660) cpu_start: ELF file SHA256:  71b36fda6e4066fa...
I (1665) cpu_start: ESP-IDF:          5.1.2
I (1669) cpu_start: Min chip rev:     v0.0
I (1672) cpu_start: Max chip rev:     v3.99 
I (1676) cpu_start: Chip rev:         v3.1
I (1681) heap_init: Initializing. RAM available for dynamic allocation:
I (1687) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
I (1692) heap_init: At 3FFBC688 len 00023978 (142 KiB): DRAM
I (1697) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
I (1703) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
I (1708) heap_init: At 4009A240 len 00005DC0 (23 KiB): IRAM
I (1714) esp_psram: Adding pool of 4082K of PSRAM memory to heap allocator
I (1722) spi_flash: detected chip: generic
I (1724) spi_flash: flash io: dio
I (1739) app_start: Starting scheduler on CPU0
I (1739) app_start: Starting scheduler on CPU1
I (1739) main_task: Started on CPU0
I (1749) esp_psram: Reserving pool of 40K of internal memory for DMA/internal allocations
I (1749) main_task: Calling app_main()
I (1749) MAIN: 



================ Start app_main =================
I (1859) MAIN: =================================================
I (1869) MAIN: ==================== Start ======================
I (1879) MAIN: =================================================
I (1879) MAIN: PSRAM size: 8388608 byte (8MB / 64MBit)
I (1889) MAIN: Total heap: 4380199 byte
I (1899) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (3899) gpio: GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2 
I (3899) cam_hal: cam init ok
I (3899) sccb: pin_sda 26 pin_scl 27
I (3899) sccb: sccb_i2c_port=1
I (3899) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (3939) camera: Detected camera at address=0x30
I (3939) camera: Detected OV2640 camera
I (3939) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4019) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4019) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4029) cam_hal: cam config ok
I (4029) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6109) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
I (6109) SDCARD: Basic R/W check started...
I (6129) SDCARD: Basic R/W check successful
I (6179) SNTP: TimeServer not defined, using default: pool.ntp.org
I (6189) SNTP: Configuring NTP Client...
I (6199) SNTP: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
I (6199) SNTP: time zone: +0100 Delta to UTC: 3600 seconds
I (6219) SNTP: The local time is unknown, starting with 1970-01-01 01:00:04
I (6229) SNTP: Once the NTP server provides a time, we will switch to that one
I (6249) MAIN: CPU frequency: 160 MHz
I (6249) OTA: Start CheckOTAUpdateCheck...
I (6269) OTA: SHA-256 for the partition table: : 9d4a2809ae90fb20709a8ca3dd3ac3462e1e73286bcd31b689060c91de06fa3a
I (6289) OTA: SHA-256 for bootloader: : 4a9573dafac5d5c79c43e5a356568332da3a99dd2e220da6ceeadc96d32708a2
I (6669) OTA: SHA-256 for current firmware: : 9a05b58889422bcde87ba674b3146abea61153f01a9043d02ab6712d64cd4a6f
I (6709) SDCARD: Folder/file presence check started...
I (6729) SDCARD: Folder/file presence check successful
I (6739) MAIN: Tag: 'v15.7.0', Release: v15.7.0 (Commit: 0d0b018+), Date/Time: 2024-02-17 00:15, Web UI: Release: v15.7.0 (Commit: 0d0b018+)
I (6759) MAIN: Reset reason: Power-on event (or reset button)
I (6769) WLANINI: SSID: Cam
I (6779) WLANINI: Password: XXXXXXXX
I (6789) WLANINI: IP-Address: 192.168.178.100
I (6789) WLANINI: Gateway: 192.168.178.1
I (6799) WLANINI: Netmask: 255.255.255.0
I (6809) WLANINI: RSSIThreshold: -75
I (6819) MAIN: WLAN config loaded, init WIFI...
I (6829) WIFI: Manual interface config -> IP: 192.168.178.100, Gateway: 192.168.178.1, Netmask: 255.255.255.0
I (6849) wifi:wifi driver task: 3ffcfe64, prio:23, stack:6144, core=0
I (6859) wifi:wifi firmware version: 91b9630
I (6859) wifi:wifi certification version: v7.0
I (6859) wifi:config NVS flash: enabled
I (6869) wifi:config nano formating: enabled
I (6869) wifi:Init data frame dynamic rx buffer num: 16
I (6869) wifi:Init static rx mgmt buffer num: 5
I (6879) wifi:Init management short buffer num: 32
I (6879) wifi:Init static tx buffer num: 16
I (6889) wifi:Init tx cache buffer num: 16
I (6889) wifi:Init static rx buffer size: 1600
I (6889) wifi:Init static rx buffer num: 16
I (6899) wifi:Init dynamic rx buffer num: 16
I (6899) wifi_init: rx ba win: 16
I (6909) wifi_init: tcpip mbox: 32
I (6909) wifi_init: udp mbox: 6
I (6909) wifi_init: tcp mbox: 6
I (6909) wifi_init: tcp tx win: 5744
I (6919) wifi_init: tcp rx win: 5744
I (6919) wifi_init: tcp mss: 1440
I (6919) wifi_init: WiFi/LWIP prefer SPIRAM
I (6929) wifi_init: WiFi IRAM OP enabled
I (6929) wifi_init: WiFi RX IRAM OP enabled
I (6939) WIFI: No DNS server, use gateway
I (6949) phy_init: phy_version 4780,16b31a7,Sep 22 2023,20:42:16
I (7029) wifi:mode : sta (08:d1:f9:6c:2d:08)
I (7029) wifi:enable tsf
I (7039) WIFI: Set hostname to: watermeter
I (7049) WIFI: Init successful
I (9449) wifi:new:<11,0>, old:<1,0>, ap:<255,255>, sta:<11,0>, prof:1
I (9449) wifi:state: init -> auth (b0)
I (9819) wifi:state: auth -> assoc (0)
I (10789) wifi:state: assoc -> run (10)
I (12019) wifi:connected with Cam, aid = 3, channel 11, BW20, bssid = 7a:83:c2:cd:ba:e3
I (12019) wifi:security: WPA2-PSK, phy: bgn, rssi: -58
I (12029) wifi:pm start, type: 1

I (12029) WIFI: Connected to: Cam, RSSI: -58
I (12039) wifi:AP's beacon interval = 102400 us, DTIM period = 1
I (12049) esp_netif_handlers: sta ip: 192.168.178.100, mask: 255.255.255.0, gw: 192.168.178.1
I (12049) WIFI: Assigned IP: 192.168.178.100
I (27059) MAIN: Device info: CPU cores: 2, Chip revision: 301
I (27069) MAIN: SD card info: Name: SDABC, Capacity: 14910MB, Free: 14878MB
I (29069) MAIN SERVER: Starting server on port: '80'
I (29069) MAIN SERVER: Registering URI handlers
I (29069) MAINCTRL: server_main_flow_task - Registering URI handlers
I (29069) OTA: Registering URI handlers
I (29069) GPIO: start GpioHandler
I (29079) GPIO: register GPIO Uri
I (29079) GPIO: server_GPIO - Registering URI handlers
I (29079) MAIN: Initialization completed successfully
I (29099) main_task: Returned from app_main()
I (29139) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (31159) LOGFILE: Set log level to ERROR
I (31169) GPIO: read GPIO config and init GPIO
I (31189) GPIO: GPIO init completed, handler is disabled
I (31189) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow!
I (40819) MAINCTRL: Autostart is not enabled -> Not starting Flow
W (50009) wifi:[ADDBA]rx delba, code:39, delete tid:0
W (65049) wifi:[ADDBA]rx delba, code:39, delete tid:0
W (80079) wifi:[ADDBA]rx delba, code:39, delete tid:0
W (95119) wifi:[ADDBA]rx delba, code:39, delete tid:0
>

**Comment by SybexX** [comment]
Da wird was mit deinen Wlaneinstellungen nicht passen >>> W (50009) wifi:[ADDBA]rx delba, code:39, delete tid:0
Im Log kannst du anhand des ersten Buchstaben erkennen um was es sich handelt:
I = Information
W = Warnung
E = Error/Fehler

Ich würde im Router den Klienten löschen und dann den Esp neustarten.
Ich weiß jetzt nicht wie der ESP seine Mac generiert, wenn Sie aber mit der im Router gespeicherten nicht übereinstimmt, lässt der Router die Verbindung nicht zu. Oder die IP wurde anderweitig an ein anderes Gerät vergeben.
